client/checkout/blocks/upe-deferred-intent-creation/payment-processor.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * External dependencies
+ */
+import {
+	PaymentElement,
+	useElements,
+	useStripe,
+} from '@stripe/react-stripe-js';
+import {
+	getPaymentMethods,
+	// eslint-disable-next-line import/no-unresolved
+} from '@woocommerce/blocks-registry';
+import { __ } from '@wordpress/i18n';
+import { useEffect, useState } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { usePaymentCompleteHandler } from '../hooks';
+import {
+	getStripeElementOptions,
+	useCustomerData,
+} from 'wcpay/checkout/utils/upe';
+import { getUPEConfig } from 'wcpay/utils/checkout';
+import { validateElements } from 'wcpay/checkout/classic/upe-deferred-intent-creation/payment-processing';
+
+const getBillingDetails = ( billingData ) => {
+	return {
+		name: ( billingData.first_name + ' ' + billingData.last_name ).trim(),
```

Comment:
```suggestion
		name: `${ billingData.first_name } ${ billingData.last_name }`.trim(),
```
🤷 Little tidier maybe? Up 2 u.